Vagrantfile

```diff
@@ -124,7 +125,27 @@ Vagrant.configure("2") do |config|
       ]
     }
     ansible.extra_vars = {
-      "pxe_server_ip_addr" => ip_addr + '/' + ip_prefix
+      "pxe_server_ip_addr" => ip_addr_server + '/' + ip_prefix
     }
   end
 end
+
+Vagrant.configure("2") do |config|
+  config.vm.hostname = 'pxe-client'
+  config.vm.box = 'centos/7'
+  config.vm.define 'pxe-client'
+  config.ssh.username = 'root'
+
+  config.vm.network "private_network",
+    auto_config: true,
+    libvirt__dhcp_bootp_file: 'pxelinux.0',
+    libvirt__dhcp_bootp_server: ip_addr_server,
+    libvirt__network_name: 'pxe-server0'
+
+  config.vm.provider :libvirt do |domain|
```

Comment:
This asks vagrant-libvirt to apply these settings to all the VMs... instead we should specify this in the host-specific way. (see other comments).